### ISSUE TITLE
require-super-in-init fails if returning the super

### ISSUE THREAD

**Issue by kellyselden** [ember-cli/eslint-plugin-ember#193]
```js
  init() {
    return this._super(...arguments);
  }
```

makes the rule fail. Removing the `return` makes it pass.

**Comment by rwjblue** [comment]
awesome, the rule is not properly identifying / allowing `ReturnStatement`s

**Comment by initram** [comment]
I think this can be closed as well, the changes that supposedly fixes this have been merged long ago.

**Comment by rwjblue** [comment]
Thanks!
